cards/src/main/resources/cards/custom/group2/set_633/core/minion_desciple_of_shitakiri.json

```diff
@@ -6,7 +6,7 @@
   "baseAttack": 1,
   "baseHp": 7,
   "rarity": "RARE",
-  "description": "When a friendly minion Deflects, gain +2 Attack.",
+  "description": "Whenever a friendly minion Deflects, gain +2 Attack.",
```

Comment:
In my experience this card is really strong so just noting it here

Reply:
I agree slightly, so I sent a message to Grand about this

Reply:
nerfed from 7 health to 6